CoreFoundation/Collections.subproj/CFArray.c

```diff
@@ -971,8 +971,10 @@ void CFArraySortValues(CFMutableArrayRef array, CFRange range, CFComparatorFunct
         result = CF_OBJC_CALLV((NSMutableArray *)array, isKindOfClass:[NSMutableArray class]);
         immutable = !result;
     } else if (CF_IS_SWIFT(CFArrayGetTypeID(), array)) {
+#if DEPLOYMENT_RUNTIME_SWIFT
```

Comment:
Should we move this up one line, such that the `else` is inside the guard too?

Reply:
It's not strictly necessary, because CF_IS_SWIFT becomes (0) in CFinternal.h:733 when DEPLOYMENT_RUNTIME_SWIFT is undefined. 

Let me know if you'd prefer it be moved, though, since it won't really hurt.